src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (props) => {
+  const { name } = props;
```

Comment:
**[Optional]**
We can destructure `name` at the argument level, that way we avoid line 5.